src/equations/compressible_euler_2d.jl

```diff
@@ -1886,6 +1886,27 @@ end
     return SVector(w1, w2, w3, w4)
 end
 
+# Transformation from conservative variables u to entropy vector ds_0/du,
+# using the specific entropy of Guermond et al. (2019): s_0 = p * rho^(-gamma) / (gamma-1).
+# Note: This is *not* the "conventional" specific entropy s = ln(p / rho^(gamma)).
+@inline function cons2entropy_spec(u, equations::CompressibleEulerEquations2D)
```

Comment:
Just noticed this... sorry: We now have `cons2entropy_spec` but it's _not_ the specific entropy. Isn't this bound to someone making an error and using the wrong one? Should we maybe just name this `cons2entropy_guermondetal`?

Reply:
That's actually a good point. I think that come from Fluxo where this routine originally used the real specific entropy but was changes later on. I'm fine with changing it.
Would you still keep the additional note here and below?

Reply:
> Would you still keep the additional note here and below?

Absolutely!

Reply:
Thanks. Done in [27ded75](https://github.com/trixi-framework/Trixi.jl/pull/1792/commits/27ded75f469a66d6632815626d373549710bf1fc)